Basic-Car-Maintenance/Shared/Dashboard/Views/ExportOptionsView.swift

```diff
@@ -9,12 +9,20 @@
 import SwiftUI
 import PDFKit
 
+enum ExportOption: String, Identifiable, CaseIterable {
+    case pdf = "PDF"
+    case csv = "CSV"
+    var id: Self { self }
+}
+
 struct ExportOptionsView: View {
     @Environment(\.dismiss) var dismiss
     @State private var selectedVehicle: Vehicle?
     @State private var isShowingThumbnail = false
     @State private var pdfDoc: PDFDocument?
     @State private var showingErrorAlert = false
+    @State private var selectedOption: ExportOption?
+    @State private var showingExporter = false
```

Comment:
```suggestion
    @State private var showingCSVGeneratorView = false
```

renaming this, and how the variable is used is this file, cause I think it feels more clear cause "showExporter" also feels like it would be used for the PDF, but it isn't